### ISSUE TITLE
We should change the art to avoid IP conflicts with the original game.

### ISSUE THREAD

**Issue by RafeHaab** [yairm210/UnCiv#226]
I love your game, even after playing it just for a day. But I couldn't help but notice that some things, specially the citizen/resource markers in the city view look exactly the same.

I'm not specially skilled in making art, but I think I could replace all you grabbed from the "other game that must not be mentioned" and help UnCiv develop into a different looking game so it doesn't get taken down from github like other games did.

Because I wouldn't be able to live without unciv.

**Comment by yairm210** [comment]
You're right, it's high time we changed the icons.
I'll get on that tomorrow b"h

**Comment by RafeHaab** [comment]
I have started to create a logo that still resembles what you have and is not copyrighted (because I made it from scratch), and we can use it upright this time if you want. It stands for "Victory", of course, which is the main goal of this game.

And it will have four layers (instead of the original three) because we are one step ahead of them, of course. I should have it ready in a few days and then I'll start with the ingame icons.
